rs/p2p/consensus_manager/src/receiver.rs

```diff
@@ -36,7 +36,7 @@ use tokio::{
 use tokio_util::sync::CancellationToken;
 use tracing::instrument;
 
-type ReceivedAdvertSender<A> = Sender<(SlotUpdate<A>, NodeId, ConnId)>;
+type InboundSlotUpdatesSender<A> = Sender<(SlotUpdate<A>, NodeId, ConnId)>;
```

Comment:
Thanks, this is a much better name than before.

What do you think about making it even more explicit what inbound means?
```suggestion
type InBoundSlotUpdatesFromPeersSender<A> = Sender<(SlotUpdate<A>, NodeId, ConnId)>;
```

Reply:
Inbound implies it is from the peers, at least the way I think about it. So adding FromPeers is a bit redundant since this is the only place where updates can come from